.travis.yml

```diff
@@ -60,6 +60,13 @@ jobs:
         use: firrtl_build
       script:
         - sbt $SBT_ARGS +unidoc
+    - stage: test
+      name: "Tests: FIRRTL (2.13)"
+      workspaces:
+        use: firrtl_build
+      script:
+        - verilator --version
+        - sbt ++2.13.2 -J-XX:LoopStripMiningIter=0 $SBT_ARGS test
```

Comment:
do you know what `-J-XX:LoopStripMiningIter=0` is about?

Reply:
It is supposed to disable an optimization in the JVM JIT, but it doesn't actually fix the Travis run as far as I can tell.
I was hopeful it might be able to fix it because there is a bug that is only fixed in OpenJDK 11.0.3 (and Travis seems to run 11.0.2) which crashes at the same instruction: https://bugs.openjdk.java.net/browse/JDK-8208275

Reply:
The flag was added in https://github.com/freechipsproject/firrtl/pull/1796/commits/a025462626452f466fbda147fab9d68c4c9d0435 which I am going to remove before we merge.